matrix/common/dynamo_handler.py

```diff
@@ -17,17 +17,26 @@ class StateTableField(Enum):
     COMPLETED_REDUCER_EXECUTIONS = "CompletedReducerExecutions"
 
 
+class OutputTableField(Enum):
+    """
+    Field names for Output table in DynamoDB.
+    """
+    REQUEST_ID = "RequestId"
+    ROW_COUNT = "RowCount"
+
+
 class DynamoHandler:
     """
     Interface for interacting with DynamoDB Tables.
     """
     def __init__(self):
         self._dynamo = boto3.resource("dynamodb", region_name=os.environ['AWS_DEFAULT_REGION'])
-        self._state_table = self._dynamo.Table(os.environ['DYNAMO_STATE_TABLE_NAME'])
+        self._state_table = self._dynamo.Table(os.getenv("DYNAMO_STATE_TABLE_NAME"))
+        self._output_table = self._dynamo.Table(os.getenv("DYNAMO_OUTPUT_TABLE_NAME"))
 
-    def init_state_table(self, request_id, num_bundles):
+    def put_state_item(self, request_id: str, num_bundles: int):
```

Comment:
Is there any reason we would need to pass the `uuid` around instead of just the string?

Reply:
string is fine